src/internal/events.ts

```diff
@@ -12,6 +12,10 @@ export const AccountCreatedEventStruct = object({
      * New account object.
      */
     account: KeyringAccountStruct,
+    /**
```

Comment:
(To match the other definitions)

```suggestion
    account: KeyringAccountStruct,

    /**
```